opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java

```diff
@@ -75,4 +75,10 @@ public void testInstallFeatureOpennmsDiscoveryDistPollerDaoMinion() {
 		installFeature("opennms-discovery-distPollerDaoMinion");
 		System.out.println(executeCommand("features:list -i"));
 	}
+	
+	@Test
```

Comment:
Sure ok. I pushed all the code changes except the test ITs. I will include blueprint tests and will update as required.